### PR TITLE
Revamp market views

### PR DESCRIPTION
**Pull request by mariano-aguero**
No issue related

### CODE REVIEW

**Comment by mariano-aguero** [comment]
Now look like this ...
Screenshot_20210107_000550

File: src/components/admin/MarketItem.vue [hunk 1]
```diff
@@ -117,36 +118,13 @@ export default {
   mounted() {
     this.$parent.$parent.$on('reload', this.reset)
   },
-  created() {
+  async created() {
```
Comment (mariano-aguero) [inline src/components/admin/MarketItem.vue:121]: I reduced all this complex chained methods using async/await

File: src/components/layouts/base/AppBar.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <router-link class="mx-5" :to="{ name: 'MyActivity' }"> Home </router-link>
       <router-link class="mx-5" :to="{ name: 'SupplyBorrow' }"> Supply/Borrow </router-link>
       <!-- <template v-if="isOwner"> -->
-      <router-link class="mx-5" :to="{ name: 'Admin' }"> Admin </router-link>
+      <router-link class="mx-5" :to="{ name: 'Status' }"> Status </router-link>
```
Comment (mariano-aguero) [inline src/components/layouts/base/AppBar.vue:10]: Mmm I don't know if this is the best name for the section, any suggestions are welcome

File: src/components/layouts/base/AppBar.vue [hunk 3]
```diff
@@ -39,7 +39,7 @@ export default {
     title() {
       if (this.$route.path === '/supplyBorrow') return 'Supply / Borrow'
       if (this.$route.path === '/myActivity') return 'My Activity'
-      if (this.$route.path === '/admin') return 'Admin Dashboard'
+      if (this.$route.path === '/status') return 'Status'
```
Comment (mariano-aguero) [inline src/components/layouts/base/AppBar.vue:42]: Idem here

File: src/middleware/market.js [hunk 4]
```diff
@@ -97,6 +97,31 @@ export default class Market {
...
+  }
+
+  async getTotalCash(formatted) {
+    // Cash is the amount of underlying balance owned by this cToken contract. One may query the total amount of cash currently available to this market.
+    const balance = await this.instance.callStatic.getCash()
+    return formatted ? ethers.utils.formatEther(balance) : balance
+  }
+
+  async getTotalSupply() {
+    // Mmmm see https://compound.finance/docs/ctokens#total-supply, probably we need to use this method, but depends of the bussines rule, we will use the sum of the total borrow and cash
```
Comment (mariano-aguero) [inline src/middleware/market.js:118]: Please take a look at this method very carefull, I don't know if the client wants the total-supply as we can check in the docs (https://compound.finance/docs/ctokens#total-supply) , or maybe just the _sum_ of the _totalCash_ and the _totalBorrow_ of the market

File: src/components/layouts/base/AppBar.vue (hunk 2 above)
Comment (pmprete) [reply src/components/layouts/base/AppBar.vue:10]: Anything is better than admin xD

File: src/components/admin/MarketItem.vue (hunk 1 above)
Comment (pmprete) [reply src/components/admin/MarketItem.vue:121]: good 👍

File: src/components/layouts/base/AppBar.vue (hunk 2 above)
Comment (pmprete) [reply src/components/layouts/base/AppBar.vue:10]: overview is also an option

File: src/middleware/market.js (hunk 4 above)
Comment (pmprete) [reply src/middleware/market.js:118]: ERC20 total supply only show minted tokens, so there will be a small diference between this and the sum, as the sum takes into account the acurred interesests that have bot yet been called.
Long story short i think it's ok what you did

**Review by pmprete — APPROVED** [review]
Awsome work Mariano

**Pull request merged**